src/sass/patterns/_data-tables.scss

```diff
@@ -19,3 +19,18 @@
     padding-right: 0;
   }
 }
+
+.table-row-targets {
+  .slds-checkbox__label {
+    padding: $spacing-medium $spacing-large;
+  }
+
+  &.slds-table_cell-buffer tr > th:first-child,
+  &.slds-table_cell-buffer tr > td:first-child,
+  &.slds-table--cell-buffer tr > th:first-child,
+  &.slds-table--cell-buffer tr > td:first-child,
+  &.slds-table th:first-child,
+  &.slds-table td:first-child {
+    padding: 0;
+  }
+}
```

Comment:
@stacyk I'm not sure this entire block is worth it -- looks like it just shifts the padding slightly? Or if you do think it's important, can you add a comment explaining what it does and why?

Default styles:

![Screenshot 2020-01-31 15 19 41](https://user-images.githubusercontent.com/552316/73571985-be065000-443d-11ea-8c98-5032d466feb8.png)

With these overrides:

![Screenshot 2020-01-31 15 19 57](https://user-images.githubusercontent.com/552316/73572000-c65e8b00-443d-11ea-9f7e-b57b673cbae8.png)

Reply:
Ill add a comment. It is making the target area wider. So anywhere you click within a row now selects the checkbox. There used to be big gaps on the left where you couldn't click and the button on the right filled most, but not all of the space.